modules/accessing-microshift-cluster-remotely.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+:_content-type: CONCEPT
+[id="accessing-microshift-cluster-remotely_{context}"]
+= Accessing the MicroShift cluster remotely
+
+Use the following procedure to access the MicroShift cluster remotely. 
+
+.Prerequisites 
+
+* You have installed the `oc` CLI. 
+
+.Procedure 
+
+. Open the Kubernetes API port in the firewall of your {op-system} machine running MicroShift by entering the following command: 
++
+[source,terminal]
+----
+$ MICROSHIFT_MACHINE=<user>@<name or IP of your MicroShift machine>
+----
+
+. Enter the following command to `SSH` into the MicroShift service and open the firewall port for `6443/tcp`:
++
+[source,terminal]
+----
+$ ssh $MICROSHIFT_MACHINE "sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload"
+----
+
+. Optional: Enter the following command to create a `~./kube/` folder if your {op-system} machine does not have one: 
++
+[source,terminal]
+----
+$ mkdir -p ~/.kube/
+----
+
+. Copy the generated `kubeconfig` file from the {op-system} machine running MicroShift to your local machine by entering the following command: 
++
+[source,terminal]
+----
+$ ssh $MICROSHIFT_MACHINE "sudo cat /var/lib/microshift/resources/kubeadmin/kubeconfig" > ~/.kube/config
+----
+
+. Replace the `server` field in your `kubeconfig` file with the name or IP of your {op-system} machine running MicroShift by entering the following command:
++
+[source,terminal]
+----
+$ sed -i "s|server: https://127.0.0.1:6443|server: https://$MICROSHIFT_MACHINE:6443|" ~/.kube/config
+----
+
+. Verify that MicroShift running by entering the following command:
```

Comment:
If we split local and remote access into different modules, I'd propse repeating this (verifying access works) to the other module, too.

Reply:
Got it. We have to split separate procedures into separate modules. Added this to `local`.